prediction_market_agent/agents/microchain_agent/app.py

```diff
@@ -171,19 +183,37 @@ def get_function_bullet_point_list(model: str) -> str:
     if not agent_is_initialized():
         model = st.selectbox(
             "Model",
-            ["gpt-3.5-turbo-0125", "gpt-4o-2024-05-13"],
+            [
+                "gpt-4-turbo",
+                "gpt-3.5-turbo-0125",
+                "gpt-4o-2024-05-13",
+            ],
             index=0,
         )
         if model is None:
             st.error("Please select a model.")
+        st.session_state.system_prompt_select = st.selectbox(
+            "Initial system prompt",
+            [p.value for p in SystemPromptChoice],
```

Comment:
See the `SystemPromptChoice` later on but the idea is, that there are two choices:

1. "plain" or "from scratch" choice, where the agent is supposed to learn everything
2. "trader" choice, where the system prompt is the same thing as was before, to start trading right away